TLSharp.Core/TelegramClient.cs

```diff
@@ -413,15 +413,15 @@ public async Task<TLAbsMessages> GetHistoryAsync(TLAbsInputPeer peer, int offset
         /// <param name="q">User or chat name</param>
         /// <param name="limit">Max result count</param>
         /// <returns></returns>
-        public async Task<TLFound> SearchUserAsync(string q, int limit = 10, CancellationToken token = default)
+        public async Task<TLFound> SearchUserAsync(string q, int limit = 10, CancellationToken token = default(CancellationToken))
```

Comment:
cancellation-related changes should be squashed with the first commit

Reply:
Done!